x-pack/plugins/security_solution/server/lists_integration/endpoint/validators/base_validator.ts

```diff
@@ -73,6 +74,15 @@ export class BaseValidator {
     }
   }
 
+  protected async validateHasPrivilege(privilege: keyof EndpointAuthz): Promise<void> {
+    if (!(await this.endpointAuthzPromise)[privilege]) {
+      throw new EndpointArtifactExceptionValidationError(
+        'Missing right privileges for this action',
```

Comment:
Maybe use the same message as our `EndpointAuthorizationError` - 

https://github.com/elastic/kibana/blob/4c5f275df4b0fc796827cbc73f401cf8fec07e10/x-pack/plugins/security_solution/server/endpoint/errors.ts#L28

Reply:
https://github.com/elastic/kibana/pull/142472/commits/e46be77bc198e12abeb137d673f4c8062bbcb438